docs/cunumeric/source/comparison/_comparison_generator.py

```diff
@@ -1,44 +1,62 @@
 import importlib
 
+import numpy
+
+blocklist = [
+    "test",
+    "add_docstring",
+    "abs",
+    "add_newdoc",
+    "add_newdoc_ufunc",
+    "alen",
+    "alltrue",
+    "bitwise_not",
+    "compare_chararrays",
+    "cumproduct",
+    "fastCopyAndTranspose",
+    "get_array_wrap",
+    "iterable",
+    "max",
```

Comment:
the same goes here. I guess I simply don't understand what this block list is doing.

Reply:
again, `max` and `min` are aliases to `amax` and `amin`:
```
>>> np.max
<function amax at 0x7f6129219670>
>>> np.min
<function amin at 0x7f6129219820>
```
unless we want to deviate from NumPy's behavior, I don't see a reason to blocklist these names.

Reply:
OK, will remove them from the blocklist